lms/views/basic_lti_launch.py

```diff
@@ -188,7 +194,64 @@ def url_configured_basic_lti_launch(self):
         LMS, which passes it back to us in each launch request. All we have to
         do is pass the URL to Via.
         """
-        return self.basic_lti_launch(self.request.parsed_params["url"])
+        groups = None
```

Comment:
This doesn't really belong to `url_configured_basic_lti_launch` but I found I would trigger the authorization flow if it was here (to get the Canvas API credentials)